content/2-templating/5-dom-ref/aurelia1/dom-ref.ts

```diff
@@ -0,0 +1,7 @@
+export class DomRef {
```

Comment:
Component name is called "InputFocus" on other frameworks.
Please stick with the same component name for better comparison.